index.php

```diff
@@ -168,6 +168,8 @@ private function cors(): void
     }
 }
 
+class FormCarterException extends Exception{}
```

Comment:
Please move it to the beginning of this file. It will be more readable.

Reply:
Ok, moved